pkg/pipelines/tekton_builder.go

```diff
@@ -72,21 +72,31 @@ func getEventListenerPath(cicdPath string) string {
 func createTrigger(gitOpsRepo string, env *config.Environment, svc *config.Service) (v1alpha1.EventListenerTrigger, error) {
 	if env.IsCICD {
 		repo, err := extractRepo(gitOpsRepo)
-		return eventlisteners.CreateListenerTrigger("ci-dryrun-from-pr", eventlisteners.StageCIDryRunFilters, repo, "github-pr-binding", "ci-dryrun-from-pr-template", eventlisteners.GitOpsWebhookSecret, env.Name), err
+		return eventlisteners.CreateListenerTrigger("ci-dryrun-from-pr", eventlisteners.StageCIDryRunFilters, repo, "ci-dryrun-from-pr-template", eventlisteners.GitOpsWebhookSecret, env.Name, []string{"github-pr-binding"}), err
 	}
 	pipelines := getPipelines(env, svc)
 	svcRepo, err := extractRepo(svc.SourceURL)
-	return eventlisteners.CreateListenerTrigger(triggerName(svc.Name), eventlisteners.StageCIDryRunFilters, svcRepo, pipelines.Integration.Binding, pipelines.Integration.Template, svc.Webhook.Secret.Name, svc.Webhook.Secret.Namespace), err
+
+	return eventlisteners.CreateListenerTrigger(triggerName(svc.Name), eventlisteners.StageCIDryRunFilters, svcRepo, pipelines.Integration.Template, svc.Webhook.Secret.Name, svc.Webhook.Secret.Namespace, pipelines.Integration.Bindings), err
 }
 
 func getPipelines(env *config.Environment, svc *config.Service) *config.Pipelines {
-	if svc.Pipelines != nil {
-		return svc.Pipelines
-	}
+	pipeline := defaultPipelines
```

Comment:
minor nitpick here, `pipeline` should really be `pipelines` because has it more than one defined...

Reply:
I changed this one . thanks:-)